pipeline_tools/dcp_utils.py

```diff
@@ -79,7 +82,9 @@ def get_auth_token(url="https://danielvaughan.eu.auth0.com/oauth/token",
                    grant_type="client_credentials"):
     """Request and get the access token for a trusted client from Auth0.
 
-    Note: The parameters and credentials here are meant to be hard coded, the authentication is purely for identifying a user it doesn't give and permissions.
+    .. note::
+
+        The parameters and credentials here are meant to be hard coded, the authentication is purely for identifying a user it doesn't give and permissions.
```

Comment:
Should this say "it doesn't give any permissions"?

Reply:
Yes, it was a typo 🐛 Nice catch!